.bin/srclib-javascript

```diff
@@ -70,7 +70,6 @@ parser.command("scan-commonjs")
           if (deps) Object.keys(deps).forEach(function(name) { allDeps.push({name: name, version: deps[name]}); });
         }
         collectDeps(pkg.package.dependencies);
-        collectDeps(pkg.package.devDependencies);
```

Comment:
It now occurs to me that `devDependencies` include things like unit test libraries that it is useful to include as an "actual dependency." I think this was the original motivation behind including them in the list of deps. Is their presence causing major issues?